src/main/java/com/parasoft/demoapp/graphql/OrderGraphQLDataFetcher.java

```diff
@@ -0,0 +1,72 @@
+package com.parasoft.demoapp.graphql;
+
+import com.parasoft.demoapp.config.WebConfig;
+import com.parasoft.demoapp.controller.ResponseResult;
+import com.parasoft.demoapp.model.industry.OrderEntity;
+import graphql.schema.DataFetcher;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+@Component
+public class OrderGraphQLDataFetcher {
+
+    private final String orderBaseUrl;
+
+    private final RestTemplate restTemplate;
+
+    private final HttpServletRequest httpRequest;
+
+    public OrderGraphQLDataFetcher(RestTemplate restTemplate, HttpServletRequest httpRequest, WebConfig webConfig) {
+
+        this.restTemplate = restTemplate;
+        this.httpRequest = httpRequest;
+        this.orderBaseUrl = "http://localhost:" + webConfig.getServerPort() +"/v1/orders";
```

Comment:
Please pull and merge, and use "HOST_WITHOUT_PORT" in GlobalPreferencesDefaultSettingsService here to be consistent. Also please help to rename this variable to "LOCALHOST".